spring-web/src/main/java/org/springframework/web/util/UriComponentsBuilder.java

```diff
@@ -440,7 +440,7 @@ public UriComponents build() {
 	/**
 	 * Build a {@code UriComponents} instance from the various components
 	 * contained in this builder.
-	 * @param encoded whether all the components set in this builder are
+	 * @param encoded asserts whether all the components set in this builder are already
```

Comment:
Likewise here, this looks accidentally included here.

Reply:
this was on purpose to prevent confusion by people reading the javadoc. Looking at issue #25323 and just different forums for questions answers there seems to be confusion about the purpose of the method. People think it's supposed to be encoding. This is to clarify it doesn't encode and the boolean passed in is to flag if it is already encoded

Reply:
Yes, I accidentally included two separate fixes in one PR. Will submit it in a new one.